README.md

```diff
@@ -1,2 +1,33 @@
-# healthy-queue-backend
-The backend for the healthy queue system
+<h1 align="center">Welcome to healthy-queue-backend 👋</h1>
+<p>
+
+  <a href="https://github.com/healthy-queue/Documentation" target="_blank">
+    <img alt="Documentation" src="https://img.shields.io/badge/documentation-yes-brightgreen.svg" />
+  </a>
+
+</p>
+
+> backend server for the healthy-queue application
+
+### 🏠 [Homepage](https://github.com/healthy-queue)
+
+### ✨ [Demo](#blank)
+
+## Install
+
+```sh
+npm install
+```
+
+## Author
+
+* Website: https://github.com/antoni909
```

Comment:
Update to authors(s) and include all project members.

Reply:
all authors added